### DISCUSSION TITLE
The red warning

### DISCUSSION THREAD

**Discussion by hyh1618** [thomaxxl/safrs#130] [General]
Hi Thomas,
Each time I start the app, it throw this warning in red color:
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
Is there other version of code? or how do I remove this?
Thanks!

**Comment by thomaxxl** [comment]
You should use another server for production deployment:
https://flask.palletsprojects.com/en/2.2.x/server/

I typically use gunicorn behind nginx,  https://flask.palletsprojects.com/en/2.2.x/deploying/gunicorn/

**Comment by hyh1618** [comment]
Hi Tomas,
using your sample, I will run:  gunicorn -w 4
"TestSAFRS:start_api("127.0.0.1",5000)"
and it fail for "fcntl not found".
